propagation/usecases.md

```diff
@@ -0,0 +1,131 @@
+the EO community as an APEx-compliant service.
+
+---
+
+## Use Case 4: Transforming a non-openEO based workflow into an APEx-compliant service and onboarding it on the APEx Algorithm Catalogue
+
+### Problem
+A project has developed a non-openEO based workflow and seeks to:
+* Enable the service for reuse as an on-demand service.
+* Make it accessible through the APEx Algorithm Catalogue.
+
+### Examples
+
+*
+
+### Solution
+This process requires the workflow developer to select one of the following options:
+* If the workflow is not compatible with cloud-based environments and requires a desktop environment for execution, the 
+project may consider using the [APEx Toolbox Cloudification](toolboxcloud.md) service. This will ensure that the workflow
+is transformed into an APEx-compatible service, enabling its onboarding onto the APEx Algorithm Catalogue through the 
+[APEx Algorithm Hosting](hosting.md) service.
+* If the workflow is already implemented in cloud-based environments or complies with the 
+[APEx Interoperability and Compliance Guidelines](../interoperability/algohosting.md), the project can utilize the 
+[APEx Algorithm Hosting](hosting.md) service for additional support in transforming the workflow into an APEx-compliant 
+on-demand service and onboarding it onto the APEx Algorithm Catalogue.
+
+### Outcome
+The workflow is successfully migrated to an APEx-compliant on-demand service and onboarded into the APEx Algorithm 
+Catalogue, enhancing its visibility and usability within the EO community as an APEx-compliant service.
+
+---
+
+## Use Case 5: Automating EO Data Processing Pipelines for Large-Scale Data Processing
+
+### Problem
+A project is currently testing and validating a data processing pipeline on a limited subset of data and aims to upscale 
+this pipeline for production, large-scale data processing.
+
+### Examples
+
+*
+
+### Solution
+The [APEx Upscaling Service](upscaling.md) provides tools and support to assist projects in automating and scaling their 
+data processing pipelines. A key prerequisite for utilizing this service is that your data processing pipeline must be 
+APEx-compliant. The [APEx Interoperability and Compliance Guidelines](../interoperability/algohosting.md) serve as an 
+excellent starting point for ensuring your pipeline meets APEx compliance. If your project requires additional assistance 
+in making the data processing pipeline APEx-compliant, the [APEx Algorithm Hosting](hosting.md) service is available to 
+provide further support.
+
+If your pipeline includes steps that are not compatible with cloud-based environments, you may consider using the 
+[APEx Toolbox Cloudification](toolboxcloud.md) service to transform those components into modular APEx-compliant services.
+
+### Outcome
+A comprehensive data processing environment is established for your project, equipped with the necessary tools and support 
```

Comment:
not sure if we'll actually establish something like this. It might be more like a service that's available to anyone + the ability to request support for processing strategies.

Reply:
OK it will be important to rephrase this bit, to set the correct expectations.